cfme/tests/configure/test_access_control.py

```diff
@@ -3081,3 +3077,29 @@ def test_aws_keypair_list(appliance, setup_provider, provider, tenant_custom_rol
     view.flash.assert_success_message('Ownership saved for selected Key Pair')
     with user:
         assert keypair.exists
+
+
+@pytest.mark.manual
+@pytest.mark.customer_scenario
+@test_requirements.rbac
```

Comment:
You don't need this marker here, since it's been added as a global marker.

Reply:
Done